rclcpp/include/rclcpp/utilities.hpp

```diff
@@ -211,7 +211,7 @@ on_shutdown(std::function<void()> callback, rclcpp::Context::SharedPtr context =
 RCLCPP_PUBLIC
 bool
 sleep_for(
-  const std::chrono::nanoseconds & nanoseconds,
+  const std::chrono::nanoseconds nanoseconds,
```

Comment:
I am bit confused here. You mention in the description that you make it a reference, but this change looks like you remove the reference? Can you elaborate your motivation behind this change?

> Making nanoseconds parameter a reference in sleep_for function

Reply:
@Karsten1987 sorry for super late response, yes, I made a mistake in a description - I corrected it now.
The motivation here is that the reference prevents the implicit conversion to nanoseconds - if I call the function with for example 1s, it would not work.

Reply:
@monidzik See my comment below about this change.